### DISCUSSION TITLE
Plot all transits vertically

### DISCUSSION THREAD

**Discussion by LucaNap** [lightkurve/lightkurve#1047] [Q&A] [answered]
Hello, I was wondering if there was a fast way to plot all the transits from a light curve vertically, similar to plot_river but with the actual transits zoomed in (phase=[-0.02, 0.02] for example), like the picture I am attaching. If not, that would be a great addition to the current plot_river function. Thank you

![immagine](https://user-images.githubusercontent.com/74565238/116395896-fbb27e00-a824-11eb-837e-fe0586734955.png)

**Comment by barentsen** [comment, marked as answer]
Thank you for the suggestion @LucaNap.  I think this is a GREAT idea and would be a fabulous feature!

To avoid making the `plot_river` function too complicated, my preference would be to add this feature as a separate new function.

An obvious name for such new function could be `plot_transits`, but this type of plot may be equally useful for plotting other periodic signals that are not transits, so I'm looking for a better name.  Any ideas?!

**Comment by LucaNap** [comment]
plot_snake
transits_chart
plot_cascade ?

I am bad with names !

**Reply by orionlee** [reply]
`LightCurveCollection.plot()` currently does something very similar. 

```python
import lightkurve as lk

# stack three folded lightcurve vertically
lk.LightCurveCollection([lc_f1, lc_f2, lc_f3]).plot(offset = 0.05);

```

![image](https://user-images.githubusercontent.com/250644/126567197-e1b3e4c1-dfec-47d9-beaf-9d14caa67e87.png)


See:
https://github.com/lightkurve/lightkurve/blob/63a959a276098c228f9333e681837b5005bca6f1/src/lightkurve/collections.py#L225-L228
